src/libraries/Microsoft.Extensions.Logging.Console/src/StringWriter.cs

```diff
@@ -0,0 +1,182 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Text;
+using System.Collections.Concurrent;
+
+namespace Microsoft.Extensions.Logging.Console
+{
+    internal class StringWriter : TextWriter
```

Comment:
>StringWriter [](start = 19, length = 12)

can we use a different name for this one? StringWriter is a public class which can conflict/confuse with this one.